roles/boot_disk/tasks/hpe.yml

```diff
@@ -0,0 +1,44 @@
+---
+- name: HPE Power ON
+  community.general.redfish_command:
+    category: Systems
+    command: PowerOn
+    baseuri: "{{ hostvars[item]['bmc_address'] }}"
+    username: "{{ hostvars[item]['bmc_user'] }}"
+    password: "{{ hostvars[item]['bmc_password'] }}"
+
+- name: HPE Eject Virtual Media (if any)
+  community.general.redfish_command:
+    category: Manager
+    command: VirtualMediaEject
+    baseuri: "{{ hostvars[item]['bmc_address'] }}"
+    username: "{{ hostvars[item]['bmc_user'] }}"
+    password: "{{ hostvars[item]['bmc_password'] }}"
+    virtual_media:
+      image_url: "{{ boot_iso_url }}"
+    resource_id: 1
+  ignore_errors: yes
+
+- name: Set HPE OneTimeBoot Hdd
+  uri:
```

Comment:
If you can't use the redfish module can you please add a comment to explain why.